### PR TITLE
87736992 fix parallel approval display

### PR DESCRIPTION
**Pull request by phirefly**
Hides approval approval actions for requesters per @cmc333333. Also included some previous feedback from @afeld.

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -22,6 +22,10 @@ def requested_carts
     self.carts.where(approvals: {role: 'requester'})
   end
 
+  def is_approver?(cart)
```
Comment (afeld) [inline app/models/user.rb]: Minor: I'd call it `approver?` or `approver_of?`.

**Comment by cmc333333** [comment]
If I read this right, it'll prevent both the requester and observers from seeing the approve/reject buttons (yay). To test the boundaries a bit more, would an approver who has already rejected/approved continue to see the buttons?

**Comment by afeld** [comment]
@cmc333333 *Good. Eye.* :clap: 

/cc https://www.pivotaltracker.com/story/show/89017762

File: app/models/user.rb (hunk 1 above)
Comment (phirefly) [inline app/models/user.rb]: I like `approver_of?` Thx!

**Comment by phirefly** [comment]
+1
For now, we are handling that here:
Parallel approvals have to be in [pending status](https://github.com/18F/C2/blob/master/app/helpers/carts_helper.rb#L26)
Linear approvals have to be pending, since we call `ordered_awaiting_approvals` [here](https://github.com/18F/C2/blob/master/app/helpers/carts_helper.rb#L34)

On that last point, I may add the guard `return false unless cart.linear?` instead of using `cart.linear?` for consistency with the previous method. Let me know what you think.

**Comment by phirefly** [comment]
So, in a nutshell, the buttons would not be visible if an approver has already rejected/approved.

**Comment by afeld** [comment]
Can we add some specs to check that the buttons don't appear in those scenarios?

**Comment by phirefly** [comment]
Updated with specs `Displaying approval actions for a parallel cart` and `Displaying approval actions for a linear cart`

**Comment by cmc333333** [comment]
I'm ready to merge unless you want to make that name change, @phirefly

**Pull request merged**